tests/AppBundle/EdgeToEdge/UserLoginTest.php

```diff
@@ -69,4 +73,76 @@ public function testUserLoginFrontend()
         $this->isSuccessful($client->getResponse());
         $this->assertTrue($crawler->filter('Anmelden')->count() === 0, 'Button "Anmelden" found');
     }
+
+    /**
+     * @dataProvider urlProvider
```

Comment:
I think they just omitted the params in the doc to focus on relevant parts for explaining. There is also no return value in doc block in the doc.

E.g. following would be more expressive when writing an unit test cause IDE gives you code completion etc. when using these params.
```PHP
/**
 * @param Raid      $raid
 * @param Character $char
 * @param string    $url
 *
 * @dataProvider urlProvider 
...
```